src/components/layout/InfoTooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import { Info } from "lucide-react";
```

Comment:
I noticed `lucide-react` also export everything as `<Name>Icon`, which I have been using as I find it clearer what it is, maybe we should align (and possibly see if we make sure code editors only suggest one or the other)?